english/work-out/weights/weights.md

```diff
@@ -3,4 +3,6 @@ thinking "I remember how easy this was in high-school...".
 
 After getting under the bar, you quickly realize you are no longer as strong as
 you were in highschool. Struggling to press the bar back up, you concede to gravity
-and sheepishly roll the bar down off yourself.
\ No newline at end of file
+and sheepishly roll the bar down off yourself.
+
+or you go with the [personal trainer](https://github.com/cvillere/create-your-own-adventure/blob/local-user/english/work-out/personal-trainer/personal_trainer.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.